src/DynamoPackagesUI/Utilities/PublishCommands.cs

```diff
@@ -23,7 +23,7 @@ internal class PublishCefHelper : CefHelper
 
         private PackageManagerViewModel packageMgrViewModel { get; set; }
 
-        public PublishCefHelper(DynamoViewModel dynamoViewModel, PackageLoader model, PackageManagerViewModel pkgManagerViewModel) : base(dynamoViewModel, model, pkgManagerViewModel)
+        public PublishCommands(DynamoViewModel dynamoViewModel, PackageLoader model, PackageManagerViewModel pkgManagerViewModel) : base(dynamoViewModel, model, pkgManagerViewModel)
```

Comment:
Why do you need DynamoViewModel to construct this command object? What is the difference between PackageManagerCommands and PublishCommands class?